app/Http/Controllers/Api/UserApiController.php

```diff
@@ -231,17 +231,32 @@ public function updateMe(LaravelRequest $request)
             return $this->error403();
 
         $myId = Auth::user()->getId();
-        return $this->update($myId);
+        $payload = $this->getUpdatePayload();
+        return $this->_update($myId, $payload);
+    }
+
+    /**
+     * @param $id
+     * @return \Illuminate\Http\JsonResponse|mixed
+     */
+    public function update($id)
+    {
+        $payload = $this->getUpdatePayload();
```

Comment:
why doing this normalization here ?
normalization should be done at client ( action before sending the json payload)
and u should add a validation payload rule like
groups => 'sometime|int_array'

Reply:
as we discussed, it stays like this